### PR TITLE
feat(tab): refine use graph and UI improvement

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

File: tab/js/default/package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "classnames": "^2.3.1",
     "msteams-react-base-component": "^3.1.0",
     "tslib": "^2.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^16.14.0",
```
Comment (eriolchan) [inline tab/js/default/package.json:12]: why do we fallback react version from 17.x to 16.x? Please check the @type version to make sure version are aligned without compile warning

File: tab/js/default/src/components/sample/lib/useGraph.js [hunk 2]
```diff
@@ -5,48 +5,38 @@ import {
...
     { auto: false }
   );
 
-  return {
-    data: data || initial.data,
-    error: error || initial.error,
-    loading: loading || initial.loading,
-    reload,
-  };
+  return data || error || loading
```
Comment (eriolchan) [inline tab/js/default/src/components/sample/lib/useGraph.js:34]: These two clause are not equivalent, is that expected?

Comment (hund030) [reply tab/js/default/src/components/sample/lib/useGraph.js:34]: Yes, we always expose the reload method, which will be trigger when user click the `authorize` button.

File: tab/js/default/package.json (hunk 1 above)
Comment (hund030) [reply tab/js/default/package.json:12]: There is peer dependencies conflict when running `npm install` in npm 7. npm 4~6 will ignore peer dependencies entirely so we didn't encounter such error before.
Refer to: https://github.blog/2021-02-02-npm-7-is-now-generally-available/#peer-dependencies
Both the @types version and js version are latest and I have tested everything works as expected.

**Review by eriolchan — APPROVED** [review]

**Pull request merged**